### PR TITLE
update ui

### PR DESCRIPTION
**Pull request by alxx88**

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -40,10 +40,15 @@
...
 <div class = "head-container">
     <div class = "container" ng-controller="authenticationCtrl as authenticationCtrl">
-        <div class = "head-title title-font">
-            <p> <a href="#ideaAddNew"><i class="fa fa-child"></i> У меня есть идея!</a></p>
+        <div class = "head-title title-font" id="IHaveIdeaDiv">
+            <p> <!--a href="#ideaAddNew"--><i class="fa fa-child"></i> У меня есть идея!<!--/a--></p>
         </div>
-        <div	 class = "href-enter">
+
+       <div style="vertical-align: middle;" id="AddIdeaDiv">
```
Comment (Hanna-Aliakseichykava) [inline src/main/webapp/index.html]: May be, it would be better to move style for this 'div' and inner 'a' to css

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Comment by LostFan** [comment]
:+1:

**Comment by Hanna-Aliakseichykava** [comment]
:+1:

**Pull request merged**